modules/bim/lib/open_project/bim/patches/work_package_board_seeder_patch.rb

```diff
@@ -42,14 +42,12 @@ def seed_bcf_board_widgets
     end
 
     def seed_bcf_board_queries
-      status_names = ['New', 'In progress', 'Resolved', 'Closed']
-      statuses = Status.where(name: status_names).to_a
-
-      if statuses.size < status_names.size
-        raise StandardError.new "Not all statuses needed for seeding a BCF board are present. Check that they get seeded."
-      end
-
-      statuses.to_a.map do |status|
+      statuses(
+        'default_status_new',
+        'default_status_in_progress',
+        'bim.default_status_resolved',
+        'default_status_closed'
```

Comment:
I thought about that, but that would reset existing translations as the key would change, and this would create some burden for translators. I can still do it if you think that's worth it.

I saw that there is a crowdin cli. I wonder if it's possible to write some kind of script that renames a key and brings along the existing translation from the old key name to the new key name. I'd like to dig this in the future as it's not the first time that I refrain from renaming a translation key for this very reason.

Reply:
I guess it is not a question of if but rather of when to move those keys. They should not stay in that file. And if we move them later it will not get better. So yes, I think they should be moved.

If you want to try to support translators by looking into the cli, be my guest.

Reply:
ok, I'll make that then